NOTICE

```diff
@@ -1,132 +1,32 @@
-Cloudberry Database
+Apache Cloudberry (Incubating)
+Copyright 2024 The Apache Software Foundation
 
-Copyright (c) 2016-2023 HashData Technology Limited. or its affiliates. All 
-Rights Reserved.
```

Comment:
As Cloudberry has been donated to ASF and the SGA has been signed, delete the HashData copyright notice directly here.